Description: Makefile fixes
 Add $(DESTDIR) to install path. Debhelper doesn't like hardcoded
 install paths.
 .
 Debhelper already installs the manpage for us. No need for second
 install in Makefile.
Author: Jonathan McCrohan <jmccrohan@gmail.com>
Date: Tue, 17 Apr 2012 13:32:20 +0100
Last-Update: 2013-03-12
---
--- nyancat-0.1+git20120401.5a88b86.orig/Makefile
+++ nyancat-0.1+git20120401.5a88b86/Makefile
@@ -33,7 +33,6 @@ distcheck: $(distdir).tar.gz
 	@echo "*** Package $(distdir).tar.gz is ready for distribution."
 
 install: all
-	install src/nyancat /usr/bin/${package}
-	gzip -9 -c < nyancat.1 > /usr/share/man/man1/nyancat.1.gz
+	install -Dm 0755 src/nyancat $(DESTDIR)/usr/bin/${package}
 
 .PHONY: FORCE all clean check dist distcheck install
